pke/kyber/kyber1024/internal/cpapke.go

```diff
@@ -125,12 +125,12 @@ func (pk *PublicKey) EncryptTo(ct, seed, pt []byte) {
 	var e2, v, m common.Poly
 
 	// Sample r, e₁ and e₂ from B_η
-	rh.DeriveNoise(seed, 0)
+	rh.DeriveNoise(seed, 0, Eta1)
 	rh.NTT()
 	rh.BarrettReduce()
 
-	e1.DeriveNoise(seed, K)
-	e2.DeriveNoise(seed, 2*K)
+	e1.DeriveNoise(seed, K, 2)
```

Comment:
Eta1?

Reply:
No, this is Eta2 which equals 2 for all parameters.

Reply:
please add a comment in the code, so we can remember this

Reply:
I've added a variable.